doc/nrf/libraries/networking/downloader.rst

```diff
@@ -187,12 +187,32 @@ The following snippet shows how to download a file using HTTPS:
 Limitations
 ***********
 
-The library requires the host server to provide a Content-Range field in the HTTP GET response header when using HTTPS with the nRF91 Series devices.
-If this header field is missing, the library logs the following error::
+The following limitations apply to this library:
 
-   <err> downloader: Server did not send "Content-Range" in response
+nRF91 Series TLS limitation
+===========================
 
- Due to internal limitations, maximum CoAP block size is 512 bytes.
+The nRF91 Series modem has a size limit for receiving TLS packages.
+The size limit depends on modem internals and is around 2 kB.
+See modem firmware release notes for details.
+The library therefore asks the server for a content-range which must be supported by the host server when using HTTPS with the nRF91 Series devices.
```

Comment:
I think the therefore can be omitted, doesn't fit well with the previous sentences
```suggestion
The library asks the server for a content-range which must be supported by the host server when using HTTPS with the nRF91 Series devices.
```

@divipillai

Reply:
Agree.